firmware-nucleo/smart-air-hockey-table/App/driver_led.c

```diff
@@ -48,35 +38,35 @@ void Driver_LED_Init()
 	LL_TIM_EnableCounter(TIM2);
 }
 
-void Driver_LED_SetColor(uint_fast8_t x, uint_fast8_t y, uint32_t color)
+void Driver_LED_SetColor(uint_fast8_t row, uint_fast8_t col, Color color)

```

Comment:
I think `row` and `col` are backwards here (correction shown below). I avoided using these terms since they're not as intuitive.

![image](https://github.com/alanssitis/smart-air-hockey-table/assets/89476587/1672d04e-26fa-4c15-af5c-47b1371356fe)